services/src/main/java/org/exoplatform/addons/gamification/rest/ChallengeRest.java

```diff
@@ -180,8 +180,8 @@ public Response getAllChallengesByUser(@ApiParam(value = "Offset of result", req
       List<ChallengeRestEntity> challengeRestEntities = new ArrayList<>();
       for (Challenge challenge : challenges) {
         List<Announcement> challengeAnnouncements = announcementService.findAllAnnouncementByChallenge(challenge.getId(),
-                                                                                                       offset,
-                                                                                                       limit);
+                                                                                                       0,
+                                                                                                       10);
```

Comment:
@GouadriaHanen If we need just 2 then no need to put 10 as limit c c @mzorai

Reply:
done!